h/templates/includes/annotation_card.html.jinja2

```diff
@@ -1,47 +1,53 @@
-{% macro annotation_card(annotation, request, incontext_link, group=none) -%}
-
+{% macro annotation_card(presented_annotation, request, tag_link) -%}
 <li class="annotation-card">
   <header class="annotation-card__header">
     <div class="annotation-card__username-timestamp">
-      <a href="{{ request.route_url('activity.user_search', username=annotation.username) }}"
+      <a href="{{ request.route_url('activity.user_search', username=presented_annotation.annotation.username) }}"
         class="annotation-card__username">
-        {{ annotation.username }}
+        {{ presented_annotation.annotation.username }}
       </a>
-      <a href="{{ request.route_url('annotation', id=annotation.id) }}"
+      <a href="{{ request.route_url('annotation', id=presented_annotation.annotation.id) }}"
         class="annotation-card__timestamp">
-        {{ annotation.updated.strftime('%d %b %Y') }}
+        {{ presented_annotation.annotation.updated.strftime('%d %b %Y') }}
       </a>
     </div>
     <div class="annotation-card__share-info">
-      {% if group %}
-        <a href="{{ request.route_url('activity.group_search', pubid=group.pubid) }}"
+      {% if presented_annotation.group %}
+        <a href="{{ request.route_url('activity.group_search', pubid=presented_annotation.group.pubid) }}"
           class="annotation-card__groupname">
             in
             {{ svg_icon('groups', 'annotation-card__groups-icon') }}
-            {{ group.name }}
+            {{ presented_annotation.group.name }}
         </a>
       {% endif %}
-      {% if not annotation.shared %}
+      {% if not presented_annotation.annotation.shared %}
         <span class="annotation-card__is-private">
           {{ svg_icon('lock', 'annotation-card__is-private-icon') }}
-          {% if not group %}
+          {% if not presented_annotation.group %}
             Only me
           {% endif %}
         </span>
       {% endif %}
     </div>
   </header>
-  {% if annotation.quote %}
+  {% if presented_annotation.annotation.quote %}
     <section class="annotation-card__quote">
-      {{ annotation.quote|safe }}
+      {{ presented_annotation.annotation.quote|safe }}
     </section>
   {% endif %}
   <section class="annotation-card__text">
-    {{ annotation.text_rendered|safe }}
+    {{ presented_annotation.annotation.text_rendered|safe }}
+  </section>
+  <section class="annotation-card__tags">
+    {% for tag in presented_annotation.annotation.annotation.tags%}
```

Comment:
> {% for tag in presented_annotation.annotation.annotation.tags%}

All the other properties of the real Annotation object are accessed via a forwarding property in the AnnotationHTMLPresenter class. I'd suggest we should do the same for tags for consistency.